e2e/dractions/actions.go

```diff
@@ -79,8 +81,8 @@ func EnableProtection(ctx types.Context) error {
 		return err
 	}
 
-	// For volsync based replication we must create the cluster namespaces with special annotation.
-	if err := util.CreateNamespaceAndAddAnnotation(ctx.Env(), ctx.AppNamespace(), log); err != nil {
+	err = createNamespaceAndAnnotation(ctx, appNamespace, log)
+	if err != nil {
 		return err
 	}
```

Comment:
Any reason not to use:

    if err := ...; err != nil {
       ...
    }

This pattern is better because we don't modify err, it visible only in the if block.

But if this is the last operation we can simplify to:

    return createNamespaceAndAnnotation(...)